gatsby-restart/src/data/docs/wormhole/payloadcreation.md

```diff
@@ -0,0 +1,312 @@
+      console.log(result);
+      }, (err) => {
+      console.log(err);
+      });
+      //  0000003301000100000000436f6d70616e69657300426974636f696e204d696e696e67005175616e74756d204d696e6572007777772e6578616d706c652e636f6d005175616e74756d204d696e657220546f6b656e73000000000100000002540be40000000000586846801e0000000000730634ca
+
+
+### `fixed`
+
+Creates the payload for a new tokens issuance with fixed supply.
+
+#### Arguments
+
+1.  ecosystem (string, required) The ecosystem to create the tokens in, must be 1
+2.  propertyPrecision (number, required) The precision of the tokens to create:\[0, 8\]
+3.  previousId (number, required) An identifier of a predecessor token (0 for new crowdsales)
+4.  category (string, required) A category for the new tokens)
+5.  subcategory (string, required) A subcategory for the new tokens)
+6.  name (string, required) The name of the new tokens to create
+7.  url (string, required) An URL for further information about the new tokens)
+8.  data (string, required) A description for the new tokens)
+9.  amount (string, required) The number of tokens to create
+
+#### Result
+
+payload `String`: the hex\-encoded payload
+
+#### Examples
+
+
+      Wormhole.PayloadCreation.fixed(1, 1, 0, "Companies", "Bitcoin Mining", "Quantum Miner", "www.example.com", "Quantum Miner Tokens", "1000000").then((result) => {
+      console.log(result);
+      }, (err) => {
+      console.log(err);
+      });
+
+      // 0000003201000100000000436f6d70616e69657300426974636f696e204d696e696e67005175616e74756d204d696e6572007777772e6578616d706c652e636f6d005175616e74756d204d696e657220546f6b656e73000000000000989680
+
+
+### `managed`
+
+Creates the payload for a new tokens issuance with manageable supply.
+
+#### Arguments
+
+1.  ecosystem (string, required) The ecosystem to create the tokens in, must be 1
+2.  propertyPrecision (number, required) The precision of the tokens to create:\[0, 8\]
+3.  previousId (number, required) An identifier of a predecessor token (0 for new crowdsales)
+4.  category (string, required) A category for the new tokens
+5.  subcategory (string, required) A subcategory for the new tokens
+6.  name (string, required) The name of the new tokens to create
+7.  url (string, required) An URL for further information about the new tokens
+8.  data (string, required) A description for the new tokens
+
+#### Result
+
+payload `String`: the hex\-encoded payload
+
+#### Examples
+
+
+      Wormhole.PayloadCreation.managed(1, 1, 0, "Companies", "Bitcoin Mining", "Quantum Miner", "www.example.com", "Quantum Miner Tokens").then((result) => {
+      console.log(result);
+      }, (err) => {
+      console.log(err);
+      });
+      // 0000003601000100000000436f6d70616e69657300426974636f696e204d696e696e67005175616e74756d204d696e6572007777772e6578616d706c652e636f6d005175616e74756d204d696e657220546f6b656e7300
+
+
+### `partiCrwoSale`
```

Comment:
Typo. => `PartiCrowSale`

Can we add an alias of this to also be the full name of the function?
`participateCrowdSale` is much more descriptive than `partiCrowSale`

Reply:
Agree. I changed `PartiCrowSale` to `participateCrowdSale` here, in `wormholecash` and in WormholeREST